Index: ../README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>### JAVA-PROJECT\n## ***DATA VALIDATOR***\n\n\n### Hexlet tests and linter status:\n[![Actions Status](https://github.com/markiMiracle/java-project-78/actions/workflows/hexlet-check.yml/badge.svg)](https://github.com/markiMiracle/java-project-78/actions)\n<a href=\"https://codeclimate.com/github/markiMiracle/java-project-78/maintainability\"><img src=\"https://api.codeclimate.com/v1/badges/f12f3b7b6916421fe69b/maintainability\" /></a>\n<a href=\"https://codeclimate.com/github/markiMiracle/java-project-78/test_coverage\"><img src=\"https://api.codeclimate.com/v1/badges/f12f3b7b6916421fe69b/test_coverage\" /></a>\n\n\n- ### ***About it:***\n  This application validates the data. \n#### It is possible to use four data formats:  \n  - StringSchema\n  - NumberSchema\n  - MapSchema\n  - ShapeSchema\n  \n\n## Setup\n\n```bash\nmake build\n```\n\n## Run\n\n```bash\nmake run\n```\n\n## Run checkstyle\n\n```bash\nmake lint\n```\n\n## Run tests\n\n```bash\nmake test\n```\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../README.md b/../README.md
--- a/../README.md	(revision 7060a7d5b2878935b1d3fa6d70581d910117b60a)
+++ b/../README.md	(date 1716206478847)
@@ -1,42 +1,4 @@
-### JAVA-PROJECT
-## ***DATA VALIDATOR***
-
-
 ### Hexlet tests and linter status:
 [![Actions Status](https://github.com/markiMiracle/java-project-78/actions/workflows/hexlet-check.yml/badge.svg)](https://github.com/markiMiracle/java-project-78/actions)
 <a href="https://codeclimate.com/github/markiMiracle/java-project-78/maintainability"><img src="https://api.codeclimate.com/v1/badges/f12f3b7b6916421fe69b/maintainability" /></a>
 <a href="https://codeclimate.com/github/markiMiracle/java-project-78/test_coverage"><img src="https://api.codeclimate.com/v1/badges/f12f3b7b6916421fe69b/test_coverage" /></a>
-
-
-- ### ***About it:***
-  This application validates the data. 
-#### It is possible to use four data formats:  
-  - StringSchema
-  - NumberSchema
-  - MapSchema
-  - ShapeSchema
-  
-
-## Setup
-
-```bash
-make build
-```
-
-## Run
-
-```bash
-make run
-```
-
-## Run checkstyle
-
-```bash
-make lint
-```
-
-## Run tests
-
-```bash
-make test
-```
